crates/fluvio-types/src/event.rs

```diff
@@ -102,7 +102,12 @@ pub mod offsets {
             self.update(self.current_value() + 1);
         }
 
+        #[deprecated = "Replace by change_listener"]
```

Comment:
Could you create a PR with only the `change_listener` changes?  @ilaborie

Reply:
See PR #2382